package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mrshmllw/smores-react",
-  "version": "2.17.1",
+  "version": "2.17.2",
```

Comment:
you might have to manually add the release notes if you pushed tags before commititng the changelog updates